File: oop.js
class Person {
    constructor(name, id) {
        this.name = name;
        this.id = id;
    }
    display() {
        console.log("Person name and id is ", this.name, " and ", this.id);
    }

}
p1 = new Person("Raian", 3);
p2 = new Person("Orpi", 7)
p1.display();
p2.display();

File: oop.py
class Student :
    def __init__(self,name,id):
        self.name=name;
        self.id=id;
    def display(self):
        print("Student name and id is", self.name, self.id);
std1= Student("Raian",3);
std2= Student("Orpi",7);
std1.display();
std2.display();